### PR TITLE
terminate sync docs

### PR DESCRIPTION
**Pull request by SushrutHarness**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/continuous-delivery/gitops/use-gitops/sync-gitops-applications.md [hunk 1]
```diff
@@ -51,3 +51,9 @@ To sync applications using the **GitOpsSync** step:
 Here is how the resources would look in Harness after the sync process is complete.
 
 ![](./static/harness-git-ops-application-set-tutorial-40.png)
+
+## Terminate Sync
```
Comment (SmritiSatya) [inline docs/continuous-delivery/gitops/use-gitops/sync-gitops-applications.md:55]: Terminate sync

File: docs/continuous-delivery/gitops/use-gitops/sync-gitops-applications.md [hunk 2]
```diff
@@ -51,3 +51,9 @@ To sync applications using the **GitOpsSync** step:
...
+
+You can terminate an in-progress sync. To do so, go to the application page for the syncing app and find the **Terminate Sync** button in the top right corner of the UI, replacing the **Sync** button when a sync is in progress.
```
Comment (SmritiSatya) [inline docs/continuous-delivery/gitops/use-gitops/sync-gitops-applications.md:57]: To terminate an in-progress sync, go to the application for the syncing app and locate the **Terminate Sync** button in the top right corner of the UI. Replace the **Sync** button when a sync is in progress.

**Review by SmritiSatya — DISMISSED** [review]

File: docs/continuous-delivery/gitops/use-gitops/sync-gitops-applications.md (hunk 1 above)
Comment (SushrutHarness) [inline docs/continuous-delivery/gitops/use-gitops/sync-gitops-applications.md:55]:
```suggestion
## Terminate sync
```

File: docs/continuous-delivery/gitops/use-gitops/sync-gitops-applications.md (hunk 2 above)
Comment (SushrutHarness) [inline docs/continuous-delivery/gitops/use-gitops/sync-gitops-applications.md:57]:
```suggestion
To terminate an in-progress sync, go to the application for the syncing app and locate the **Terminate Sync** button in the top right corner of the UI. Replace the **Sync** button when a sync is in progress.
```

**Review by SmritiSatya — APPROVED** [review]

**Pull request merged**